### PR TITLE
Add FAQ entry on how to add GoCardless bank to limited history list.

### PR DESCRIPTION
**Pull request by psybers**
Provides steps to contribute to the GoCardless list of banks with limited (90 day) historical data.

### CODE REVIEW

File: docs/advanced/bank-sync.md [hunk 1]
```diff
@@ -133,3 +133,17 @@ In the free tier, you can sync 50 times per month. If you sync to two different
 For more information, see the [Bank Account Data API Usage](https://bankaccountdata.zendesk.com/hc/en-gb/articles/11528933493916-Bank-Account-Data-API-Usage-how-is-your-usage-number-calculated)
 topic in the GoCardless FAQ.
 
+**What if my bank only supports 90 days of historical data?**
+If your bank limits the amount of historical date, Actual may need to know about this.  You might need to add your bank to the list of banks with limited history in the actual-server code.  Here's how you can do that:
+
+1. Read the instructions about integrating with GoCardless: https://github.com/actualbudget/actual-server/blob/master/src/app-gocardless/README.md
+2. Toward the top is a link to a Google Docs. Find your bank's ID in that document.
```
Comment (RubenOlsen) [inline docs/advanced/bank-sync.md:140]: - Either remove the "a" in _a Google Docs_, or _Toward the top is a link to the Google Docs containing data points for the GoCardless integration._
- _Find_ => _Locate_
- _in that_ => _in the_

File: docs/advanced/bank-sync.md [hunk 2]
```diff
@@ -133,3 +133,17 @@ In the free tier, you can sync 50 times per month. If you sync to two different
...
+3. Fork the Actual server repository: https://github.com/actualbudget/actual-server/fork
+4. Edit the file `src/app-gocardless/bank-factory.js`.
```
Comment (RubenOlsen) [inline docs/advanced/bank-sync.md:142]: Between 3 and 4, there should be instruction on creating a branch, right?

File: docs/advanced/bank-sync.md [hunk 3]
```diff
@@ -133,3 +133,17 @@ In the free tier, you can sync 50 times per month. If you sync to two different
...
+5. Add your bank's ID (from the Google Docs) to the `BANKS_WITH_LIMITED_HISTORY` list.
+6. Commit your changes and push to your fork.
```
Comment (RubenOlsen) [inline docs/advanced/bank-sync.md:144]: Se comment regarding adding a branch to the fork.

File: docs/advanced/bank-sync.md [hunk 4]
```diff
@@ -133,3 +133,17 @@ In the free tier, you can sync 50 times per month. If you sync to two different
...
+7. Create a pull request to the main Actual server repository.
+8. Add a release note in `upcoming-release-notes/` describing your change (see [writing good release notes](/docs/contributing/#writing-good-release-notes)).
+9. Commit your changes and push to your fork.
```
Comment (RubenOlsen) [inline docs/advanced/bank-sync.md:147]: Ditto from #6

File: docs/advanced/bank-sync.md [hunk 5]
```diff
@@ -133,3 +133,17 @@ In the free tier, you can sync 50 times per month. If you sync to two different
...
+
+Once reviewed, the maintainers will comment on the pull request and merge it if it is acceptable.  The change would then be available in the next release of the software.
```
Comment (RubenOlsen) [inline docs/advanced/bank-sync.md:149]: _it is_ can be removed without losing clarity as it makes the sentence slightly shorter.

**Review by RubenOlsen — CHANGES_REQUESTED** [review]

File: docs/advanced/bank-sync.md (hunk 2 above)
Comment (psybers) [reply docs/advanced/bank-sync.md:142]: That is optional. If people want to, they can.

**Review by youngcw — APPROVED** [review]

**Pull request merged**